src/tests/ftest/object/same_key_different_value.py

```diff
@@ -176,63 +159,57 @@ def test_array_to_single_value(self):
         for i in range(3):
             try:
                 # create an object and write array value data into it
-                obj = self.container.write_an_array_value(array_value_data,
-                                                          dkey, akey,
-                                                          obj_cls=1)
+                obj = self.container.write_an_array_value(array_value_data, dkey, akey, obj_cls=1)
```

Comment:
(pylint-no-member) Instance of 'TestContainer' has no 'write_an_array_value' member
(pylint-no-member) Instance of 'list' has no 'write_an_array_value' member